src/Stratis.Bitcoin.Networks/StratisRegTest.cs

```diff
@@ -62,7 +62,8 @@ public StratisRegTest()
 
             var bip9Deployments = new StratisBIP9Deployments()
             {
-                [StratisBIP9Deployments.ColdStaking] = new BIP9DeploymentsParameters(0, 0, 999999999)
+                // Always active on StratisRegTest.
```

Comment:
Please clarify this - 'always active' implies that the deployment is immediately active, which is not what the parameters are currently set to do (timeout would need to be `-1` for that).

Reply:
Good spot. Lets improve on that:
`
[StratisBIP9Deployments.ColdStaking] = new BIP9DeploymentsParameters(1, BIP9DeploymentsParameters.AlwaysActive, 999999999)
`

Reply:
Ah yes, it is actually the start time that is meant to be `-1`, my bad. There is an incorrect comment in the `BIP9DeploymentsParameters` class.